tests/helper/test_helper.py

```diff
@@ -190,13 +190,13 @@ def test_powerset(iterable, map_func, expected_powerset):
 
 @pytest.mark.parametrize('muted_streams, muted_topics, vary_in_unreads', [
     ([99], [['Some general stream', 'Some general unread topic']], {
-        'all_msg': 9,
+        'all_msg': 8,
         'streams': {99: 1},
         'unread_topics': {(99, 'Some private unread topic'): 1},
         'all_mentions': 0,
     }),
     ([1000], [['Secret stream', 'Some private unread topic']], {
-        'all_msg': 11,
+        'all_msg': 8,
```

Comment:
If these two values vary, is this really a bugfix rather than refactor?

Reply:
This is testing `classify_unread_counts`, which as per old behavior, did not update `all_msgs` for muted streams. This was done instead in the UI while generating stream buttons. 
This commit moved that part into `classify_unread_counts`, hence there's a change in the values returned.
The overall external behavior did not change, so I'm not sure if it's correct to call it a bugfix